tests/cupy_tests/fallback_mode_tests/test_fallback.py

```diff
@@ -0,0 +1,226 @@
+
+    # Existing function
+    """
+    This test is going to fail as argmin exist in cupy,
+    if used `check_func_returning_array`.
+    Because, cupy returns 0-d array and numpy returns scalar value.
+    """
+
+    def test_argmin_zero_dim_array_vs_scalar(self):
+
+        a = ([
+            [13, 5, 45, 23, 9],
+            [-5, 41, 0, 22, 4],
+            [2, 6, 43, 11, -1]
+        ],)
+
+        cupy_args = (cupy.array(a),)
+        numpy_args = (numpy.array(a),)
+
+        kwargs = {}
+
+        # getting via fallback_mode (cupy)
+        x = getattr(fb, 'argmin')(*cupy_args, **kwargs)
+
+        # getting via native numpy
+        y = getattr(numpy, 'argmin')(*numpy_args, **kwargs)
+
+        assert int(x) == y
+
+    def test_linalg_module(self):
+        self.check_cupy_module('linalg')
+
+    def test_random_module(self):
+        self.check_numpy_module('random')
+
+    def test_matrixlib_module(self):
+        self.check_numpy_module('matrixlib')
+
+    def test_vectorize(self):
+
+        def function(a, b):
+            if a > b:
+                return a - b
+            return a + b
+
+        actual = numpy.vectorize(function)([1, 2, 3, 4], 2)
+        expected = fb.vectorize(function)([1, 2, 3, 4], 2)
+
+        assert isinstance(actual, numpy.ndarray)
+
+        # ([1,2,3,4], 2) are arguments to numpy.vectorize(function),
+        # not numpy.vectorize
+        # returns as numpy.ndarray
+        assert isinstance(expected, numpy.ndarray)
+
+        numpy.testing.assert_array_equal(cupy.asnumpy(expected), actual)
+
+    def test_notifications_enabled(self):
+
+        import contextlib
+        from io import StringIO
+
+        saved_stdout = StringIO()
+        with contextlib.redirect_stdout(saved_stdout):
+            res = fb.nanargmin([1, 2, 3])  # NOQA
+
+        output = saved_stdout.getvalue().strip()
+        assert output == "'nanargmin' not found in cupy, falling back to numpy"
+        assert len(saved_stdout.getvalue().splitlines()) == 1
+
+    def test_notifications_disabled(self):
+
+        import contextlib
+        from io import StringIO
+
+        saved_stdout = StringIO()
+        with contextlib.redirect_stdout(saved_stdout):
+            set_notifications(False)
+            assert not notifications()
+            res = fb.nanargmax([1, 2, 3])  # NOQA
+
+        output = saved_stdout.getvalue().strip()
+        assert output == "Notifications are Disabled"
+        assert len(saved_stdout.getvalue().splitlines()) == 1
+        set_notifications(True)
+        assert notifications()
+
+    def test_module_not_callable(self):
+
+        # numpy()
+        self.assertRaises(TypeError, fb)
+
+        # numpy.linalg()
+        self.assertRaises(TypeError, fb.linalg)
+
+        # numpy.linalg._cupy_module()
+        self.assertRaises(TypeError, fb.linalg._cupy_module)
+
+    def test_numpy_scalars(self):
+
+        # numpy.inf
+        assert fb.inf == numpy.inf
```

Comment:
`assert fb.inf is numpy.inf` seems better for the test.

Reply:
Okay, I'll change that.